lib/drivers/medtronic/medtronicDriver.js

```diff
@@ -972,14 +972,17 @@ module.exports = function (config) {
               callback(null, 'eot');
             });
         },
+        /*
```

Comment:
Is this commented out code expected to be a part of this review?

Reply:
Yep. I'm not sure if it will resolve the issue in the card specified in the comment, as the issue is very intermittent. So my reasoning is to comment it out (sending the 'X' is not necessary, as it's used as a wakeup call, which is now performed by the EOT sent just before it) and if it works I can later delete it as part of the card in the comment.